src/TributeRange.js

```diff
@@ -355,14 +355,22 @@ class TributeRange {
         }
     }
 
-    lastIndexWithLeadingSpace (str, char) {
+    lastIndexWithLeadingSpace (str, trigger) {
```

Comment:
I think there's quite a wide usage of 'chart' rather than 'trigger' around, so at some point the maintainers may have to look at renaming them all to avoid confusion.

Reply:
Did you mean "char" (not 'chart')?

Reply:
Yes, "char", sorry. I've been working with too much chart code, both navigational charts _and_  D&D charts.